apps/web/src/serviceWorker.ts

```diff
@@ -0,0 +1,149 @@
+// This optional code is used to register a service worker.
```

Comment:
So I'm not too familiar, but this isn't getting spun as a prod web-app, right (it would be kind of cool to do this in the future). This may be another one of those things that we can get rid of for now.